BridgeAppSDK/SBASubtaskStep.swift

```diff
@@ -55,7 +55,7 @@ public class SBASubtaskStep: ORKStep {
     
     func substepIdentifier(identifier: String) -> String? {
         guard let range = identifier.rangeOfString("\(self.subtask.identifier).") else { return nil }
-        let stepRange = Range(start: range.endIndex, end: identifier.endIndex)
+        let stepRange = range.endIndex ..< identifier.endIndex
```

Comment:
Double checked to make sure that unit tests really covered this and added a test to ensure that they both do the same thing.